ui/src/templates/forms/widget.hbs

```diff
@@ -20,6 +20,10 @@
                     {{> inputs/checkbox value=commonProperties.useDuration id=useDuration name="useDuration" title="Set a duration?" helpText="Select to provide a specific duration for this Widget" }}
                     {{> inputs/number value=commonProperties.duration id=duration name="duration" title="Duration" helpText="The duration of the widget in seconds" visibility='{ "field": "useDuration", "type": "eq", "value": "1" }' }}
                     {{> inputs/dropdown value=commonProperties.enableStat options=(arr (obj name='Off' title='Off') (obj name='On' title='On') (obj name='Inherit' title='Inherit')) optionsTitle="title" optionsValue="name" id=enableStat name="enableStat" title="Enable Stats Collection?" helpText="Enable the collection of Proof of Play statistics for this Widget. Ensure that ‘Enable Stats Collection’ is set to ‘On’ in the Display Settings." }}
+
+                    {{#if repeatDataActive}}
+                        {{> inputs/checkbox value=commonProperties.isRepeatData id=isRepeatData name="isRepeatData" title="Is Repeat Data?" helpText="// TODO: Write something here!" }}
```

Comment:
Repeat Items?
Once all data items have been placed into slots, any empty slots will be filled with items from the start.